src/py21cmfast/src/heating_helper_progs.c

```diff
@@ -1310,3 +1319,116 @@ float zmax(float z, int n){
     denom = 1 - pow(n, -2);
     return (1+z)*num/denom - 1;
 }
+
+//Function to read \deltaE
```

Comment:
If I understand it correctly, this is Delta E of eqs. 18 and 19 in Reis et al. which is tabulated into a function of Ts, Tk and optical depth. Can I just confirm, this is in units of erg right?

Reply:
Yes.